doc/manual/expressions/builtins.xml

```diff
@@ -405,6 +405,15 @@ stdenv.mkDerivation { … }
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>sha256</term>
```

Comment:
I think we should support SRI hashes instead.

Reply:
The patch now provides a single `hash` argument which expects an SRI hash. In case of a hash mismatch, SRI hashes will be printed and it's documented how to generate an SRI hash of a git repo (`nix to-sri --type sha256 $(nix-prefetch-git ...)`).